src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -76,7 +76,13 @@ trait LibraryServiceSupport extends LazyLogging {
       AttributeName.withDefaultNS("workspaceId") -> AttributeString(workspace.workspaceId),
       AttributeName.withDefaultNS("authorizationDomain") -> AttributeValueList(workspace.authorizationDomain.map(group => AttributeString(group.membersGroupName.value)).toSeq)
     )
-    val fields = attrfields ++ idfields
+
+    val tags: Option[Attribute] = workspace.attributes.get(AttributeName.withTagsNS())
+
+    val fields = attrfields ++ idfields ++ (tags match {
+      case Some(t) => Map(AttributeName.withTagsNS() -> t)
+      case None => Map()
+    })
```

Comment:
for code cleanliness: `attrfields` and `idfields` are both already `Map`s. I would make the tag code follow the same pattern by moving the match earlier:

```
    val tagfields = workspace.attributes.get(AttributeName.withTagsNS()) match {
      case Some(t) => Map(AttributeName.withTagsNS() -> t)
      case None => Map.empty[AttributeName, Attribute]
    }

    val fields = attrfields ++ idfields ++ tagfields
```